ocs_ci/ocs/fiojob.py

```diff
@@ -264,21 +282,69 @@ def delete_fio_data(fio_job_file, delete_check_func):
 
 def workload_fio_storageutilization(
     fixture_name,
-    target_percentage,
     project,
     fio_pvc_dict,
     fio_job_dict,
     fio_configmap_dict,
     measurement_dir,
     tmp_path,
+    target_percentage=None,
+    target_size=None,
     with_checksum=False,
 ):
     """
     This function implements core functionality of fio storage utilization
-    workload fixture. This is necessary because we can't parametrize single
+    workload fixtures. This is necessary because we can't parametrize single
     general fixture over multiple parameters (it would mess with test case id
     and polarion test case tracking).
+
+    It works as a workload fixture, as understood by
+    :py:mod:`ocs_ci.utility.workloadfixture` module.
+
+    When ``target_percentage`` is specified, the goal of the fixture is to fill
+    whatever is left so that total cluster utilization reaches the target
+    percentage. This means that in this mode, number of data written depends
+    on both total capacity and current utilization. If the current storage
+    utilization already exceeds the target, the test is skipped.
+
+    On the other hand with ``target_size``, you can specify the size of data
+    written by fio directly.
+
+    Args:
+        fixture_name (str): name of the fixture using this function (for
+            logging and k8s object labeling purposes)
+        project (): project in which the Job is deployed
```

Comment:
project (str):

Reply:
good point, this is actually some kind of ocp object ...

Reply:
fixed